diesel/src/mysql/backend.rs

```diff
@@ -4,16 +4,17 @@ use backend::*;
 use query_builder::bind_collector::RawBytesBindCollector;
 use super::query_builder::MysqlQueryBuilder;
 
-#[derive(Debug, Copy, Clone)]
+#[derive(Debug, Copy, Clone, Hash, PartialEq, Eq)]
 pub struct Mysql;
 
-#[allow(missing_debug_implementations, missing_copy_implementations)]
+#[allow(missing_debug_implementations)]
```

Comment:
Why put that attr above and the derive below the doc string? No, wait, let me guess: Because doc strings are in fact `#[doc=…]` and you wanted it sorted alphabetically? 😄